### PR TITLE
docs: added cross-chain comparison

### PR DESCRIPTION
**Pull request by Marty-Q**
This document adds an additional page highlighting the cross-chain process of Zenrock's solutions. It compares how classic cross-chain solutions work and how zenrock's MPC solution behaves differently.

### CODE REVIEW

File: docs/docs/cross-chain.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### Cross-Chain Problem 
+
+The number of blockchains in the space is rapidly growing and so is the fragementation of liquidity, use cases and developer attention. This fragmentation makes it difficult for users to navigate the space and for developers to build on top of it. 
+
+Zenrock aims to solve this fragmentation by providing an MPC-based protocol that allows users to send and receive messages on and across different blockchains with the same level of security as the blockchain they originated from where the users can chose their security level based on their risk appetite. 
+
+### Cross-Chain Solutions
+
+The most common way to communicate cross chain is with a centralized bridge. While these bridges are useful, they come with their own set of risks such as single points of failure, loss of funds due to exploits and human error, and the need for users to trust the bridge with their assets. On top of that, it often includes many and larger smart contract interactions which can take a long time to execute and and therefore higher transaction fees.
```
Comment (AndreiMitrea) [inline docs/docs/cross-chain.md:17]:
```suggestion
The most common way to communicate cross chain is with a centralized bridge. While these bridges are useful, they come with their own set of risks such as single points of failure, loss of funds due to exploits and human error, and the need for users to trust the bridge with their assets. On top of that, it often includes many and larger smart contract interactions which can take a long time to execute and therefore higher transaction fees. 
```

File: docs/docs/cross-chain.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+One key can be reused across multiple chains because the public key is derived into blockchain-specific addresses. An `secp256k1` key can be derived into an EVM-address starting with `0x` or a Zenrock address starting with `zen`, and many more. This makes the UX of using Zenrock seamless and native compared to other solutions. While the private key is secretly stored across multiple MPCs, the public keys are stored on `zrChain` and `zrSign` where they can be derived into multiple blockchain-specific addresses.
+
+What is signed by the MPCs is solely determined by the user and can be as simple as a message or as complex as a swap. The MPCs do not know nor care about what is in the message as long as the request is approved by authorized and required parties within Zenrock's policy manager. This concept leaves developers the freedom to chose and craft the unsigned transactions as they see fit for their use cases. Zenrock itself is also working on solutions to facilitate MPC-based keys to provide compelling use cases like zenBTC.
```
Comment (AndreiMitrea) [inline docs/docs/cross-chain.md:29]:
```suggestion
What is signed by the MPCs is solely determined by the user and can be as simple as a message or as complex as a swap. The MPCs do not know nor care about what is in the message as long as the request is approved by authorized and required parties within Zenrock's policy manager. This concept leaves developers the freedom to choose and craft the unsigned transactions as they see fit for their use cases. Zenrock itself is also working on solutions to facilitate MPC-based keys to provide compelling use cases like zenBTC.
```

File: docs/docs/cross-chain.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+<div style={{maxWidth: "900px", margin: "0 auto"}}>
+
+![Cross-Chain Process](../static/img/cross-chain-process.png)
+
+</div>
+
+### Encouraging Developers to Build on Top of Zenrock
+
+With this universal, cheap, and fast cross-chain solution, Zenrock is encouraging developers to build on top of either `zrChain` in CosmWasm (Rust) or `zrSign` in Solidity. We are providing strong support for developers to build on top of our solutions and to provide a seamless experience for their users.
```
Comment (AndreiMitrea) [inline docs/docs/cross-chain.md:41]:
```suggestion
With this universal, cheap, and fast cross-chain solution, Zenrock is encouraging developers to build on top of either `zrChain` in CosmWasm (Rust) or `zrSign` in Solidity. We are providing strong support for developers to build on top of our solutions and to offer a seamless experience for their users. 
```

File: docs/docs/cross-chain.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+### How Zenrock MPC is Different
+
+Zenrock's MPC-generated keys can be used on supported blockchains natively without the need for a bridge. This means that unsigned transactions, once they are signed by the MPCs, can be sent directly to the destination chain's RPC where they are executed as a standard transaction and which can be found in the standard block explorers. 
+
+One key can be reused across multiple chains because the public key is derived into blockchain-specific addresses. An `secp256k1` key can be derived into an EVM-address starting with `0x` or a Zenrock address starting with `zen`, and many more. This makes the UX of using Zenrock seamless and native compared to other solutions. While the private key is secretly stored across multiple MPCs, the public keys are stored on `zrChain` and `zrSign` where they can be derived into multiple blockchain-specific addresses.
```
Comment (AndreiMitrea) [inline docs/docs/cross-chain.md:27]: > While the private key is secretly stored across multiple MPCs
For me it reads like multiple MPCs are storing the full private key, while the private key is actually split in shares allocated to MPCs.

Comment (Marty-Q) [reply docs/docs/cross-chain.md:27]: It's a bit more complex than just splitting and sharing. I'd recommend to leave it as is. We have a more detailed explanation on the mpc page

**Review by AndreiMitrea — APPROVED** [review]

**Review by wjlosh — APPROVED** [review]

**Pull request merged**